waspc/examples/todoApp/src/server/actions.ts

```diff
@@ -1,8 +1,9 @@
 import HttpError from '@wasp/core/HttpError.js'
+import { Task } from '@wasp/entities/'
+import { AuthenticatedAction } from '@wasp/types'
 import { getSomeResource } from './serverSetup.js'
 
-
-export const createTask = async (task, context) => {
+export const createTask: AuthenticatedAction<[Task]> = async (task, context) => {
```

Comment:
If understand correctly, this list `[Task]` should the same list as you defined in the `.wasp` file?

Reply:
Yes, it's a whole thing. Check this comment and follow links for the full story: https://github.com/wasp-lang/wasp/pull/945#issuecomment-1384501234